help/sources/tutorials/ui/create/data-partners/acxiom-prospecting-data-import.md

```diff
@@ -0,0 +1,154 @@
+  * **Description** - Optional description for logging.
+* **Alerts** - Adobe Experience Platform can produce event-based alerts which users can subscribe to, these options all a running dataflow to trigger these.  [Alert overview](../../../../../observability/alerts/overview.md)
+  * **Sources Dataflow Run Start** - Issues an alert when the dataflow starts.
+  * **Sources Dataflow Run Success** - Issues an alert when the dataflow ends without error.
+  * **Sources Dataflow Run Failure** - Issues an alert when the dataflow ends with a failure status.
+<br>    ![Dataset details](../../../../images/tutorials/create/acxiom-prospect-suppression-data-sourcing/image-source-dataset-details.png)
+
+## Mapping ##
+
+Defines interface of source columns to associated schema columns on Adobe.  Platform automatically provides intelligent recommendations for auto-mapped fields based on the target schema or dataset that you selected. You can manually adjust mapping rules to suit your use cases or fix any duplicated mapping fields to clear any errors. [Field mapping in the UI](../../../../../data-prep/ui/mapping.md)
+
+* **Search source fields** - Entering text into this field will restrict the existing map fields, showing only those that contain that substring.
+* **"All fields" filter fields** - Drop down  restricting the existing map fields, showing only those that match the status indicated in the drop down.  Options include:
+  * **All fields (Default)** - Display all mapping
+  * **Fields with errors** - Show only those that have validation errors
+  * **Required fields** - Show only those that are required
+  * **Identity fields** - Show only those that are defined as identity fields
+  * **Mapped fields** - Show only those that have been mapped
+  * **Unmapped fields** - Show only those that have not been mapped
+* **New field type**
+  * **Add new field** - New fields can be appended, allowing a user to duplicate a source column to a different target field, or adding fields that were previously discarded
+  * **Add calculated field** - Allows field manipulation though scripting.  [Field mapping functions in the UI](../../../../../data-prep/functions.md)
+* **Import Mapping** - This allows mapping to be imported from previous dataflow configurations.  When selected a list will be presenting with the dataflows, their last run status, date of processing, as well as the source used.
+* **Preview Data** - This option will create an example of the mapping as presented.  A filter option allows to restrict the result to specific columns or outlining columns presenting errors.
+* **Validate** - This repeats the validation of the current mapping presented, ensuring that fields qualify to the schema selected.
+* **"Clear all" mapping** - Resets the mapping, removing all entries.
+<br>![Mapping](../../../../images/tutorials/create/acxiom-prospect-suppression-data-sourcing/image-source-mapping.png)
+
+## Scheduling ##
```

Comment:
- Remove extra ##
- Revise to "Schedule your dataflow ingestion"

Reply:
Cleaned up extra ## and changed section title to suggested